test/pretty.test.js

```diff
@@ -8,236 +8,240 @@ const writer = require('flush-write-stream')
 const { once } = require('./helper')
 const pino = require('../')
 
-test('can be enabled via exported pino function', async ({ isNot }) => {
-  var actual = ''
-  const child = execa(process.argv[0], [join(__dirname, 'fixtures', 'pretty', 'basic.js')])
-
-  child.stdout.pipe(writer((s, enc, cb) => {
-    actual += s
-    cb()
-  }))
-  await once(child, 'close')
-  isNot(actual.match(/\(123456 on abcdefghijklmnopqr\): h/), null)
-})
-
-test('can be enabled via exported pino function with pretty configuration', async ({ isNot }) => {
-  var actual = ''
-  const child = execa(process.argv[0], [join(__dirname, 'fixtures', 'pretty', 'level-first.js')])
-
-  child.stdout.pipe(writer((s, enc, cb) => {
-    actual += s
-    cb()
-  }))
-  await once(child, 'close')
-  isNot(actual.match(/^INFO.*h/), null)
-})
-
-test('can be enabled via exported pino function with prettifier', async ({ isNot }) => {
-  var actual = ''
-  const child = execa(process.argv[0], [join(__dirname, 'fixtures', 'pretty', 'pretty-factory.js')])
-
-  child.stdout.pipe(writer((s, enc, cb) => {
-    actual += s
-    cb()
-  }))
-
-  await once(child, 'close')
-  isNot(actual.match(/^INFO.*h/), null)
-})
-
-test('does not throw error when enabled with stream specified', async ({ doesNotThrow }) => {
-  doesNotThrow(() => pino({ prettyPrint: true }, process.stdout))
-})
-
-test('throws when prettyPrint is true but pino-pretty module is not installed', async ({ throws, is }) => {
-  // pino pretty *is* installed, and probably also cached, so rather than
-  // messing with the filesystem the simplest way to generate a not found
-  // error is to simulate it:
-  const prettyFactory = require('pino-pretty')
-  require.cache[require.resolve('pino-pretty')].exports = () => {
-    throw Error(`Cannot find module 'pino-pretty'`)
-  }
-  throws(() => pino({ prettyPrint: true }))
-  try { pino({ prettyPrint: true }) } catch ({ message }) {
-    is(message, 'Missing `pino-pretty` module: `pino-pretty` must be installed separately')
-  }
-
-  require.cache[require.resolve('pino-pretty')].exports = prettyFactory
-})
-
-test('can send pretty print to custom stream', async ({ is }) => {
-  const dest = new Writable({
-    objectMode: true,
-    write (formatted, enc) {
-      is(/^INFO.*foo\n$/.test(formatted), true)
-    }
+const isWin = process.platform === 'win32'
```

Comment:
no changes to tests, only "if" block added.

Reply:
Couldn’t they just be avoided by doing a `process.exit(0)` before the tests if the platform is Windows?

Reply:
Does tap execute all tests in separate processes? Otherwise there's a risk to abort test process prematurely. I thought about doing `return`, but you can't return from non-function.

Reply:
Yes, each test file is run in its own process.

Reply:
Change implemented, thanks!